svthalia/concrexit
# Copy product name for order items

Closes #1655

### Summary
Copies the product name for to order items, to allow changing product names

### How to test
1. Have an order with products
2. Change a product name or delete the product
3. The product name should still exist for the order item
4. Also test the API
